src/lib/consent-components/components/consent-modal-navigation/consent-modal-navigation.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="navigation-icons">
+  <i
+    class="material-icons"
+    [ngStyle]="{ visibility: showArrow ? 'visible' : 'hidden' }"
+    (click)="navigateBack()"
+    >keyboard_backspace</i
```

Comment:
This "> ... </i" formating is confusing me.

Reply:
That's what prettify done. Weird indeed.

Reply:
Now should be better